ICS-ICAL Sync/Code.gs

```diff
@@ -20,13 +20,14 @@ var sourceCalendars={
 
 // Currently global settings are applied to all sourceCalendars.  
 
-var howFrequent = 1; //What interval (minutes) to run this script on to check for new events
+var howFrequent = 20; //What interval (minutes) to run this script on to check for new events
 var addEventsToCalendar = true; //If you turn this to "false", you can check the log (View > Logs) to make sure your events are being read correctly before turning this on
+var modifyExistingEvents = true; // If you turn this to "false", any event in the feed that was modified after being added to the calendar will not update
 var removeEventsFromCalendar = true; //If you turn this to "true", any event in the calendar not found in the feed will be removed.  
 var createMissingCalendar=true;  // If the calendar name is not found we create it instead of throwing an error, only if the sourceUrl works.
 var addAlerts = true; //Whether to add the ics/ical alerts as notifications on the Google Calendar events
 var descriptionAsTitles = false; //Whether to use the ics/ical descriptions as titles (true) or to use the normal titles as titles (false)
-var defaultDuration = 60; //Default duration (in minutes) in case the event is missing an end specification in the ICS/ICAL file
+var defaultDuration = 30; //Default duration (in minutes) in case the event is missing an end specification in the ICS/ICAL file
```

Comment:
What is the argument for making this change? I don't see a reason to change the default value of `defaultDuration`

Reply:
No good reason.  Just forgot i changed it.